From 890341694c407fed77b817ccef1f1f70c3506378 Mon Sep 17 00:00:00 2001
From: Pablo Greco <pablo@fliagreco.com.ar>
Date: Fri, 12 Jul 2019 21:41:38 -0300
Subject: [PATCH] Leave more space (4MB) for uboot before the first partition

Uboot is getting bigger, and in some cases, it doesn't even fit in 1MB
(like BeagleBoneBlack).
Start the first partition at 4MB to leave more space.
---
 appcreate/partitionedfs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appcreate/partitionedfs.py b/appcreate/partitionedfs.py
index b738b80..b2b0413 100644
--- a/appcreate/partitionedfs.py
+++ b/appcreate/partitionedfs.py
@@ -90,7 +90,7 @@ class PartitionedMount(Mount):
                 p['type'] = 'primary'
                 p['num'] = d['numpart']
             if d['offset'] == 0:
-                d['offset'] = 1
+                d['offset'] = 4
             p['start'] = d['offset']
             d['offset'] += p['size']
             d['partitions'].append(n)
-- 
2.21.0

